### ISSUE TITLE
wrong autocomplete when there is only one function in module

### ISSUE THREAD

**Issue by slashmili** [slashmili/alchemist.vim#52]
If a module has only one function  : 

```
defmodule TryOut.Multi do
  def run_me do
  end
end
```

By entering `TryOut.M` and pressing `<c-x><c-o>` it autocomplete  to `TryOut.run_me`

**Comment by slashmili** [comment]
another case:

```
Phoe<auto>
Phoenix.C<auto>
Channel
```

**Comment by slashmili** [comment]
Fixed in #55
